model/src/main/resources/db-update/20011-from-2.1.1-to-2.2.0.sql

```diff
@@ -0,0 +1,52 @@
+--
+-- JBoss, Home of Professional Open Source.
+-- Copyright 2014-2020 Red Hat, Inc., and individual contributors
+-- as indicated by the @author tags.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+-- NCL-6695: Add Operation table
+BEGIN;
+
+    CREATE TABLE operation (
+        id bigint NOT NULL,
+        starttime timestamptz NOT NULL,
```

Comment:
@janinko this should be probably nullable, as we said the operation will be started after it is inserted